auth_user_case_insensitive/i18n/zh_CN.po

```diff
@@ -0,0 +1,41 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * auth_user_case_insensitive
+#
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-05-17 21:08+0000\n"
+"PO-Revision-Date: 2017-05-17 21:08+0000\n"
+"Last-Translator: OCA Transbot <transbot@odoo-community.org>, 2017\n"
+"Language-Team: Chinese (China) (https://www.transifex.com/oca/teams/23907/"
+"zh_CN/)\n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#. module: auth_user_case_insensitive
+#: code:addons/auth_user_case_insensitive/hooks.py:0
+#, python-format
+msgid "Conflicting user logins exist for `%s`"
+msgstr ""
+
+#. module: auth_user_case_insensitive
+#: model:ir.model.fields,field_description:auth_user_case_insensitive.field_res_users__login
+msgid "Login"
+msgstr ""
+
+#. module: auth_user_case_insensitive
+#: model:ir.model.fields,help:auth_user_case_insensitive.field_res_users__login
+msgid "Used to log into the system. Case insensitive."
+msgstr ""
+
+#. module: auth_user_case_insensitive
+#: model:ir.model,name:auth_user_case_insensitive.model_res_users
+msgid "Users"
+msgstr "用户"
```

Comment:
Could include lt.po

Reply:
Added lt.po, not sure how great my translation is though, as there is no good translation for the phrase `case sensitive`.